azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
```

Comment:
What do you think about separating this part into its own method and add a unit test for it?